Sources/Views/Cells/MediaMessageCell.swift

```diff
@@ -84,14 +84,13 @@ open class MediaMessageCell: MessageContentCell {
     
     /// Handle tap gesture on contentView and its subviews.
     open override func handleTapGesture(_ gesture: UIGestureRecognizer) {
-        let touchLocation = gesture.location(in: self)
-        
-        switch true {
-        case imageView.frame.contains(touchLocation):
-            delegate?.didTapImage(in: self)
-        default:
-            break
+        let touchLocation: CGPoint = convert(gesture.location(in: self), to: imageView)
+
+        guard imageView.bounds.contains(touchLocation) else {
+            super.handleTapGesture(gesture)
```

Comment:
Did you test all the cases associated with this? We might want to write some tests against this change

Reply:
I didn't test all the cases. I am assuming it's working properly on the superclass.